src/sentry/api/bases/organization_events.py

```diff
@@ -8,7 +9,20 @@ class OrganizationEventsEndpointBase(OrganizationEndpoint):
 
     def get_snuba_query_args(self, request, organization):
         params = self.get_filter_params(request, organization)
+        query = request.GET.get('query')
+
         try:
-            return get_snuba_query_args(query=request.GET.get('query'), params=params)
+            snuba_args = get_snuba_query_args(query=query, params=params)
         except InvalidSearchQuery as exc:
             raise OrganizationEventsError(exc.message)
+
+        # TODO(lb): remove once boolean search is fully functional
```

Comment:
We'll probably actually convert this to the `advanced-search` feature once we release